sugarscape.py

```diff
@@ -1103,6 +1107,14 @@ def verifyConfiguration(configuration):
         removeDecisionModels = uniqueAgentDecisionModels[maxColors:]
         configuration["agentDecisionModels"] = [i for i in configuration["agentDecisionModels"] if i not in removeDecisionModels]
 
+    # Ensure experimental group is properly defined or otherwise ignored
+    if configuration["experimentalGroup"] == "":
+        configuration["experimentalGroup"] = None
+    elif configuration["experimentalGroup"] not in ["depressed", "female", "male", "sick"]:
```

Comment:
`None` should be included in the values to check. If `experimentalGroup` is left out of the config (e.g. in examples), debug will print `Cannot provide separate log stats for experimental group None. Disabling separate log stats.` The `None` value was correctly provided by `sugarscape.py` — this is intended behavior and not a user error.

Reply:
Done.